test/cpp/naming/cancel_ares_query_test.cc

```diff
@@ -260,7 +260,7 @@ TEST(CancelDuringAresQuery, TestFdsAreDeletedFromPollsetSet) {
   grpc_pollset_set_destroy(fake_other_pollset_set);
 }
 
-void TestCancelDuringActiveQuery(bool set_short_query_timeout) {
+void TestCancelDuringActiveQuery(const char* query_timeout_setting) {
```

Comment:
Using a string for this seems fairly messy.  Please use an enum instead.

Reply:
done